code/game/machinery/iv_drip.dm

```diff
@@ -219,7 +219,8 @@
 		return
 	if(isanimal(usr))
 		var/mob/living/simple_animal/user_animal = usr
-		return user_animal.canUseTopic()
+		if (!user_animal.canUseTopic())
```

Comment:
Last quip--I don't think you need to be checking for animals at all. Everything implements `canUseTopic`.